changelog.md

```diff
@@ -358,14 +361,14 @@
 - Handle empty modifier list correctly in `vmq_webhooks` (#339).
 - Handle client_id and mountpoint modifiers correctly in `vmq_webhooks` (#332).
 - Fix vmq-admin session show when multiple filters are applied
-- Fix bug where an aborted connection handshake caused queued messages not 
+- Fix bug where an aborted connection handshake caused queued messages not
   being migrated properly to remote node.
 
 ## VERNEMQ 1.0.0
 
 - To make the `vmq-admin` tool more consistent, the following changes have been
   made:
-  
+
```

Comment:
Could you revert all the white space changes? I've configured my editor to only remove trailing white space on the line I actually edit, maybe your editor has a similar feature?

Reply:
Yep, i need to fix that, what editor do you use BTW?

Reply:
emacs - for good and for worse ;)

Reply:
For good I guess, it has the best Erlang support, I just cannot unlearn VIM :)

Reply:
VIM has pretty good Erlang support as well I think - I'm sure @dergraf would agree